flask_of_oil/opaque_validator.py

```diff
@@ -142,4 +142,5 @@ def validate(self, token):
 
         return {"subject": introspect_response['sub'],
                 "scope": introspect_response['scope'],
+                "cnf": introspect_response.get("cnf", ""),
```

Comment:
good

Reply:
Maybe we should revert this as it doesn't make any difference right now. We should find a more generic approach to send back all claims to the request.

We only use the `active`, `scope` and `subject` from opaque_validator.validate (and the jwt_validator.validate method) in oauth_filter.filter.

`active` is checked for True/False, `scope` is used to authorize the request comparing with the configured scopes and `subject` is saved into g.user

Reply:
ya, something generic like we did in the Java API when we updated it for the licensing service. That's what we need.